src/runner.jl

```diff
@@ -234,12 +257,23 @@ function Base.showerror(io::IO, ex::PkgTestError, bt; backtrace=true)
     printstyled(io, ex.msg, color=Base.error_color())
 end
 
-function test(pkgs::Vector{AbstractString}; logfilepath::String=pwd(), kwargs...)
+function test(pkgs::Vector{<:AbstractString}; 
+              logfilename::Vector{<:AbstractString}=[pkg * "_testlog.xml" for pkg in pkgs], 
```

Comment:
Should this be plural?
Or should this like not be an argument and it should just always us this value?

Reply:
Yeah I wasn't sure what was best to do with this. If we always use this value it would affect the 1 package use case, and then the xml would change name if the package name changed, so I think it needs to only default to this when `test` is called directly with a vector of packages.

If it was plural you'd have two kwargs, one being `logfilename` and one being `logfilenames` and I didn't know if that would get confusing. Or do you think that would be okay? The method errors would probably help with any confusion.

We could rename it "logfilenaming" or something similar so it works for single or double, but loses clarity.

I think plural is the best option perhaps?

Reply:
hmms maybe singluar is best.
Though removing it for both the single package case and the vector of packages case i think might also be fine.
If it always is a predictable name, then user can always to `mv` in post-processing to give it the name of choice

Reply:
I'm tempted to leave them in, as it will default to sensible behaviour but if some really does want to change the name(s) they can. For example if they already had a `testlog.xml` in their repository